src/dmd/typesem.d

```diff
@@ -56,6 +56,8 @@ import dmd.target;
 import dmd.tokens;
 import dmd.typesem;
 
+enum LOGDOTEXP = 0;         // log ::dotExp()
```

Comment:
Why did you move this declaration?

Reply:
because it was a leftover from creating the dotExp visitor. You will notice that it is not used at all in mtype.d . I did some debugging in typesem, so I needed the LOGDOTEXP definition